recipe/build.sh

```diff
@@ -1,8 +1,15 @@
 #!/bin/bash
 
+source activate "${CONDA_DEFAULT_ENV}"
+
 chmod +x configure
 
-./configure --prefix=$PREFIX --enable-cxx
+if [ "$(uname)" == "Darwin" ];
+then
+    ./configure --prefix=$PREFIX --enable-cxx --build=x86_64-apple-darwin
```

Comment:
From the thread

>Not a known issue, but also not of concern. Fat binary builds don't need to work on everyone's machines, just on some machine that someone producing binaries has access too. The fat build process is pretty fragile, and not terribly well maintained.

It's understandable that this would fail on travis. This might work in some other Mac.